app/models/article.rb

```diff
@@ -1,5 +1,22 @@
 class Article < ApplicationRecord
+  #validations
   validates :title, presence: true
```

Comment:
add length validation to title

Reply:
validation implemented, please check latest commit.